src/stories/Form.stories.tsx

```diff
@@ -139,18 +152,21 @@ function Template(args) {
 
 /**
  * Story to exhibit the native event handlers for TextInput in the Form Component
- * @param {Object} args
- * @returns {JSX}
  */
-function WithNativeEventHandler(args) {
+function WithNativeEventHandler(args: FormProviderProps<typeof ONYXKEYS.FORMS.TEST_FORM>) {
     const [log, setLog] = useState('');
 
     // Form consumes data from Onyx, so we initialize Onyx with the necessary data here
     NetworkConnection.setOfflineStatus(false);
-    FormActions.setIsLoading(args.formID, args.formState.isLoading);
-    FormActions.setErrors(args.formID, args.formState.error);
+    FormActions.setIsLoading(args.formID, !!args.formState?.isLoading);
     FormActions.setDraftValues(args.formID, args.draftValues);
 
+    if (args.formState?.error) {
+        FormActions.setErrors(args.formID, {error: args.formState.error as MaybePhraseKey});
+    } else {
+        FormActions.clearErrors(args.formID);
+    }
```

Comment:
See [this comment](https://github.com/Expensify/App/pull/37820/files#r1516366451)